test/integration/component/test_browse_volumes.py

```diff
@@ -2028,6 +2028,10 @@ def test_01_Browser_volume_Life_cycle_tpath(self):
         """
         Test Browser_volume_Life_cycle - This includes upload volume,attach to a VM, write data ,Stop ,Start, Reboot,Reset  of a VM, detach,attach back to the VM, delete volumes  
         """
+        if self.hypervisor.lower() == 'hyperv':
```

Comment:
Can we skip only the resize volume test case with if condition and run test case for other operations?